### ISSUE TITLE
Suggestion for implementing event queues

### ISSUE THREAD

**Issue by ewidl** [ERIGrid2/JRA-3.1-api#13]
I propose to provide at least a suggestion on how to implement the event queue. Something like "keep the previous 100 events" or "keep events for at least 5 minutes".

**Comment by stv0g** [comment]
Good idea. I think we can agree on a minimally supported queue length. Furthermore, I would say that the API must provide the real queue size via the `/info` endpoint
